sources/packages/backend/workflow/test/2021-2022/parttime-assessment/eligibility/parttime-assessment-2021-2022-eligibility-BCAG.e2e-spec.ts

```diff
@@ -103,4 +104,69 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-BCAG
       });
     }
   });
+
+  it("Should determine provincialAwardNetBCAGAmount when awardEligibilityBCAG is true", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataCRAReportedIncome = 20001;
+    assessmentConsolidatedData.studentDataRelationshipStatus = "married";
+    assessmentConsolidatedData.studentDataIsYourSpouseACanadianCitizen =
+      YesNoOptions.Yes;
+    assessmentConsolidatedData.partner1CRAReportedIncome = 22999;
+    // Public institution
+    assessmentConsolidatedData.institutionType = InstitutionTypes.BCPublic;
+    // Act
+    const calculatedAssessment = await executePartTimeAssessmentForProgramYear(
+      PROGRAM_YEAR,
+      assessmentConsolidatedData,
+    );
+    // Assert
+    // calculatedDataTotalFamilyIncome = 20001 + 22999
+    // awardEligibilityBCAG is true
+    // federalAwardBCAGAmount is greater than or equal to 100
+    expect(calculatedAssessment.variables.calculatedDataTotalFamilyIncome).toBe(
+      43000,
+    );
+    expect(calculatedAssessment.variables.calculatedDataFamilySize).toBe(2);
+    expect(calculatedAssessment.variables.awardEligibilityBCAG).toBe(true);
+    expect(
+      calculatedAssessment.variables.federalAwardBCAGAmount,
+    ).toBeGreaterThan(100);
+    expect(calculatedAssessment.variables.provincialAwardNetBCAGAmount).toBe(
+      Math.min(
+        calculatedAssessment.variables.calculatedDataTotalRemainingNeed3,
+        Math.min(
+          calculatedAssessment.variables.dmnPartTimeAwardAllowableLimits
+            .limitAwardBCAGAmount,
+          calculatedAssessment.variables.federalAwardBCAGAmount,
+        ),
+      ),
+    );
+    expect(calculatedAssessment.variables.provincialAwardNetBCAGAmount).toBe(
+      1000,
+    );
+  });
+
+  it("Should determine provincialAwardNetBCAGAmount as zero when awardEligibilityBCAG is false", async () => {
```

Comment:
I feel this test is same as the one in line 83 except the assert 
```ts
expect(calculatedAssessment.variables.provincialAwardNetBCAGAmount).toBe(0);
```

![image](https://github.com/bcgov/SIMS/assets/54600590/20f3ad4f-3a47-4276-9dd2-b21c487b2384)

Reply:
The two tests are not exactly the same. The on in the image determines BCAG eligibility, but line 150 determines `provincialAwardNetBCAGAmount`. Do you want me to move the assertion of `finalProvincialAwardNetBCAGAmount` to inside the test with line 150?